theme/models.py

```diff
@@ -110,6 +110,57 @@ class IconBox(Orderable):
         help_text="Optional, if provided clicking the box will go here.")
 
 
+class QuotaMessage(models.Model):
+    # warning_content_prepend will prepend the content to form a warning message to be emailed
+    # to the user and displayed when the user is logged in; enforce_content_prepend will prepend
+    # the content to form an enforcement message to inform users when they are over allocated quota
+    warning_content_prepend = models.TextField(default='Your quota for HydroShare resources is '
+                                                       '{allocated}{unit} in zone {zone}. You '
+                                                       'currently have resources that consume '
+                                                       '{used}{unit}, {percent]% of your quota. '
```

Comment:
@pkdash good catch. I modified the message from admin panel, so I don't see any error when testing the warning message. Will correct the typo.